apps/web/.env.example

```diff
@@ -0,0 +1,45 @@
+# [[E2E Tests]]
```

Comment:
Not sure if we want this here, didn't notice the mail stuff there either. If it already existed it was likely a mistake from myself 😓